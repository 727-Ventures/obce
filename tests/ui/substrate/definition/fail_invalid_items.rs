#[obce::definition]
pub trait Trait {
    const INVALID_ITEM: u32 = 123;
}

fn main() {}
