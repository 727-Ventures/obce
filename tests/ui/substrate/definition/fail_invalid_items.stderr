error: only methods are supported in trait definitions
 --> tests/ui/substrate/definition/fail_invalid_items.rs:3:5
  |
3 |     const INVALID_ITEM: u32 = 123;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
